src/Microsoft.Data.Sqlite.Core/Microsoft.Data.Sqlite.Core.csproj

```diff
@@ -0,0 +1,43 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\build\common.props" />
+  <PropertyGroup>
+    <AssemblyName>Microsoft.Data.Sqlite</AssemblyName>
+    <RootNamespace>$(AssemblyName)</RootNamespace>
+    <PackageId>$(MSBuildProjectName)</PackageId>
+    <Description>SQLite implementation of the System.Data.Common provider model.
```

Comment:
Nothing was handling `<Summary>`. NuGet [doesn't support it](https://docs.microsoft.com/en-us/nuget/schema/msbuild-targets)... (I was getting the warning even with it.)